src/modules/select/wallets/alphawallet.ts

```diff
@@ -0,0 +1,75 @@
+import { mobileWalletInstallMessage } from '../content'
+import {
+  WalletModule,
+  Helpers,
+  InjectedWithBalanceOptions
+} from '../../../interfaces'
+
+import alphawalletIcon from '../wallet-icons/icon-alphawallet'
+
+function alphawallet(options: InjectedWithBalanceOptions): WalletModule {
+  const { preferred, label, iconSrc, svg, rpcUrl } = options
+
+  return {
+    name: label || 'AlphaWallet',
+    svg: svg || alphawalletIcon,
+    iconSrc,
+    wallet: async (helpers: Helpers) => {
+      const { getProviderName, getAddress, getNetwork, getBalance } = helpers
+      const alphawalletProvider =
+        (window as any).ethereum ||
+        ((window as any).web3 && (window as any).web3.currentProvider)
+
+      const isAlphaWallet = getProviderName(alphawalletProvider) === 'AlphaWallet'
+      let createProvider
+
+      if isAlphaWallet && rpcUrl) {
```

Comment:
@JamesSmartCell Should there be an opening parenthesis here? 
```suggestion
      if (isAlphaWallet && rpcUrl) {
```